fabric-network/src/events.ts

```diff
@@ -8,34 +8,34 @@ import { FilteredBlock, FilteredTransaction, Endorser, EventInfo } from 'fabric-
 import Long = require('long');
 
 export interface BlockEvent {
-	getType(): 'filtered' | 'full' | 'private';
-	getBlockNumber(): Long;
+	readonly type: 'filtered' | 'full' | 'private';
```

Comment:
is it worth updating the index.js jsdoc file to say that these properties are readonly? or is that pointless?